magento/merchdocs#1162: Stores | Expand a methods of Using a Custom Admin URL (#1181)
magento/merchdocs: src/stores/store-urls-custom-admin.md

   {:.bs-callout-info}
   This method allows you to change the Admin Path, but not the Admin URL.

>    {:.bs-callout-info}

Reviewer: This new callout appears to clash with the one above it. Do we need to be clear here that "Admin path" and "Admin URL" are different? This is suggesting that one is set by the other.

Reviewer 2: This should probably be a "tip", since it lets Cloud customers know there is a different way to set this. Many things in the Admin UI are read-only for Cloud-hosted Magento. If we made a note each time, it would reduce the quality of the user guide.

Reviewer 2: Some of this request was documented in #1098

>    On Magento Cloud, a custom admin path may be set via the **ADMIN_URL** Cloud [variable](https://devdocs.magento.com/cloud/env/environment-vars_magento.html#admin-url) in the Cloud UI. From the technical side, the specified value is added to `app/etc/env.php` within the environment build.

Reviewer 2: This is getting a little techie for the user guide. I would keep it brief.

```suggestion
   On Magento Cloud, a custom admin path may be set using the **ADMIN_URL** [variable](https://devdocs.magento.com/cloud/env/environment-vars_magento.html#admin-url) in the Cloud UI.
```

Reviewer 2: I edited this suggestion after the review so that it was clear that the link was taking them to the Cloud guide.

Author: Thank you, lady !
I'll implement your suggestion very soon :)

   {:.bs-callout-info}
   Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.

> ## Method 3: Change the Admin Path via CLI

Reviewer 2: Since method 2 indicates the command line also, I suggest changing this header to be more specific.

```suggestion
## Method 3: Change the Admin Path using the Magento CLI
```

> Magento provides a command to change the Admin Path - `setup:config:set`.
> 
> 1. To change the admin `frontName`, from the Magento root, run:

Reviewer 2: If there is only one step, then it is best not to use an ordered list.

```suggestion
You can use the Magento CLI `setup:config:set` command to change the Admin Path. The following example uses the `--backend-frontname` option to change the path from the Magento root to a new Admin path:
```

    ```terminal
    bin/magento setup:config:set --backend-frontname="backend_front_name"
    ```

>     where `backend_front_name` is the new Admin Path.
> 
>     {:.bs-callout-info}
>     This command will update `backend` > `frontName` configuration option in the `app/etc/env.php` file.

Reviewer 2: I consolidated the information, so this can be cleaned up a bit.

```suggestion
This command updates the `backend` > `frontName` configuration option in the `app/etc/env.php` file.
```
